packages/sdk/src/services/seeder.service.js

```diff
@@ -170,10 +170,17 @@ module.exports = {
       preferredPort: this.settings.config.swarm_port
     }
 
+    let secret = this.settings.config.secret
+    if (!secret) {
+      this.logger.warn('It is recommended to setup a secret value in the config. See more here: https://github.com/geut/permanent-seeder#secret')
+      secret = 'defaultS3cret!'
+    }
+
```

Comment:
Do I understand correctly this would only activate if the secret is actively removed from the default `.toml` by the sysadmin?

Reply:
if the secret is not present in the settings. The idea behind this is to support former permanent-seeder instances. These are already running and may not update their settings file but we can salt this with a default value. Then, the users will see the warning indicating they can change the secret value by updating their settings file.